Flippers/src/main/java/io/doist/recyclerviewext/flippers/EmptyRecyclerFlipper.java

```diff
@@ -9,29 +9,46 @@
 import androidx.recyclerview.widget.RecyclerView;
 
 /**
- * Handles a {@link RecyclerView}'s empty view, similar to the automatic control provided by {@link ListActivity} for
- * {@link ListView}.
+ * Handles a {@link RecyclerView}'s empty view, similar to the automatic control provided by {@link
+ * ListActivity} for {@link ListView}.
  */
-public class EmptyRecyclerFlipper extends Flipper {
+public class EmptyRecyclerFlipper {
```

Comment:
Now it doesn't extend `Flipper` anymore. Can we avoid it? Otherwise client's code can become broken it it casts `EmptyRecyclerFlipper` to `Flipper` :)

Reply:
I was trying to avoid subclassing and refactored to composing. But it would break clients code, so I've added back the subclassing and added warnings of deprecation on the methods and on the README :)

Reply:
> I was trying to avoid subclassing in favor of composing.

I think choice subclassing vs composing was made when this library was implemented. I think it's late to change this 😅